src/Symfony/Bridge/Twig/Extension/SerializeExtension.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\Serializer\Serializer;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ */
+class SerializeExtension extends AbstractExtension
+{
+    private $serializer;
+
+    public function __construct(Serializer $serializer)
```

Comment:
We should use the runtime system - https://symfony.com/doc/current/templating/twig_extension.html#creating-lazy-loaded-twig-extensions - for laziness.

Example from core: https://github.com/symfony/symfony/blob/f0c3bc9f320a26f55be9371bf9012410d399d3d9/src/Symfony/Bundle/FrameworkBundle/Resources/config/security_csrf.php#L45-L47

Reply:
done